AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs

```diff
@@ -171,17 +171,12 @@ public void ClickToProgress(By element, By newElement, int timeout = 20)
             {
                 try 
                 {
-                   Console.WriteLine($"loop {i}");
                     JavascriptClick(element);
                     Thread.Sleep(TimeSpan.FromSeconds(1)); 
                     Driver.WaitUntilVisible(newElement, 1);
-                    Console.WriteLine($"new page found in loop {i}");
                     return; 
                 }
-                catch (Exception e)
-                {
-                    Console.WriteLine($"new page NOT found in loop {i}");
-                }
+                catch (Exception e) {}
```

Comment:
It's not normally good practice to have empty exceptions. In this case any exception caught is ignored. I think either remove the try catch and have an action in the catch

Reply:
That's intentional, that exception needs be caught and ignored so to continue looping.  I'll change it to two catches, one for webdrivertimeoutexception and another for no suchelementexception.